src/rohiniBDec19/Assignment2/Program5.java

```diff
@@ -0,0 +1,31 @@
+package rohiniBDec19.Assignment2;
+
+/*5)Write a java program to find second largest element in an array of integers? */
+
+public class Program5 {
+	
+	int temp;
+	void findNum(int arr[]) {	
+		for(int i=0;i<arr.length;i++) {
+			for(int j=i+1;j<arr.length;j++)
```

Comment:
good but how you will doing without using any algorithm.

Reply:
Here i am sorting array like  small number at 1st and large at end hence retrieving sencibd last element